client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -542,17 +539,31 @@ export class CheckoutThankYou extends Component<
 				);
 			}
 
+			// Keep the AtomicStoreThankYouCard for the Woo Mobile App
+			if ( isWcMobileApp() ) {
```

Comment:
This doesn't look safe to me, `AtomicStoreThankYouCard` is doing email verification and sets the CTA url to `siteWooCommerceWizardUrl`, can we use the StandardCheckoutThankYou inside AtomicStoreThankYouCard and keep the same behavior?

We're also losing the "Create your store!" CTA text. "Lets work on the site" doesn't sound right for the ecommerce plan.

Reply:
This PR also sets the CTA to the same URL as `siteWooCommerceWizardUrl`.

As for email verification, I tested, and with the current flow, the user will get redirected to another email verification page when clicking on the CTA `siteWooCommerceWizardUrl`
![admin page](https://github.com/Automattic/wp-calypso/assets/6586048/bed05c2e-4bad-4e6d-a8e6-38568ff78693)
Once they verify their email and click on connect jetpack, they will go to the store setup page
![setup page](https://github.com/Automattic/wp-calypso/assets/6586048/7d55ce9e-2a61-4229-a311-e544de862ede)

Summarizing the flow for unverified emails:

How this PR currently changes the flow:
1. Thank You page
2. Click on CTA
3. Verify Email
4. Connect Jetpack (this might be the confusing step)
5. Start setting up the store

The current production flow:
1. Thank You card
2. Verify Email
3. Click on CTA
4. Start setting up the store

![ecommerce](https://github.com/Automattic/wp-calypso/assets/6586048/7d945556-ab4b-447e-bc2c-9c48addbe1f4)


Any ideas on how we could incorporate the old flow into the new design
cc: @JanaMW27

Reply:
> How this PR currently changes the flow:
> 
> Thank You page
> Click on CTA
> Verify Email
> Connect Jetpack (this might be the confusing step)
> Start setting up the store
> 
> The current production flow:
> 
> Thank You card
> Verify Email
> Click on CTA
> Start setting up the store

How did we add the Jetpack connection step?

> Any ideas on how we could incorporate the old flow into the new design

I'd need to dig in to really judge this but I think the new design needs to support the old design's functionality other wise the only option is to revert to the "new" thank you component approach and let them control the flow and we just control the basic design outline.

I guess you could change something higher up to but I don't know how that would work.

What are you thinking?

Reply:
Oh, also this option looked ok if I'm remembering it correctly: 

> can we wrap StandardCheckoutThankYou inside AtomicStoreThankYouCard and keep the same behavior?

Reply:
> How did we add the Jetpack connection step?

I didn't add anything in particular. It was... just there 😅 when I tested the flow with the PR.
Maybe it would be helpful if we have more people to test the flow within this PR to confirm.